grouper/api/handlers.py

```diff
@@ -46,22 +46,20 @@ def get(self, name=None):
         with self.graph.lock:
             if not name:
                 return self.success({
-                    "users": sorted(list(self.graph.users) + list(self.graph.disabled_users.keys())),
+                    "users": sorted(list(self.graph.users) + \
```

Comment:
Is the trailing `\` necessary? I thought as long as you're within a parenthesis, you can just drop it.